### ISSUE TITLE
Custom lib documentation

### ISSUE THREAD

**Issue by ChialiT** [Enaleia/eas-lib#2]
Ensure other engineers can easily find which lib and function to use and why

**Comment by ChialiT** [comment]
@unidwell should we update the README at least?

**Comment by unidwell** [comment]
Yes. I've updated it to reflect the code changes and linked the examples into the README.
